static/js/mediastream/webrtc.js

```diff
@@ -218,7 +287,7 @@ function($, _, PeerCall, PeerConference, PeerXfer, PeerScreenshare, UserMedia, u
 
 	WebRTC.prototype.processReceivedMessage = function(to, data, type, to2, from) {
 
-		if (!this.started) {
+		if (!this.started && type !== "Conference") {
```

Comment:
Same as above, the types are all strings for now, so it should be done everywhere in a separate PR.